rj_gameplay/rj_gameplay/tactic/pass_tactic.py

```diff
@@ -50,19 +50,17 @@ def __call__(self,
             else:
                 return 1
 
-
-
 class Pass(tactic.ITactic):
     """
     A passing tactic which captures then passes the ball
     """
 
-    def __init__(self, target_point:np.ndarray):
+    def __init__(self, target_point:np.ndarray, passer_cost:role.CostFn, receiver_cost:role.CostFn):
         self.target_point = target_point
         self.pivot_kick = tactic.SkillEntry(pivot_kick.PivotKick(target_point = target_point, chip=False, kick_speed=4.0))
         self.receive = tactic.SkillEntry(receive.Receive())
-        self.receiver_cost = Receiver_cost(target_point)
-        self.Passer_cost = Passer_cost()
+        self.receiver_cost = receiver_cost
+        self.Passer_cost = passer_cost
```

Comment:
This capitalization is off. Also, are the classes on lines 17 & 36 really supposed to be capitalized like that? I thought it was `passer_cost` and `receiver_cost` for the classes that have a __call__

Reply:
```suggestion
        self.passer_cost = passer_cost
```